<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="StartButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>47</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>93</width>
     <height>28</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>93</width>
     <height>28</height>
    </size>
   </property>
   <property name="text">
    <string>出发</string>
   </property>
  </widget>
  <widget class="QComboBox" name="StrategyComboBox">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>210</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>费用最少</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>耗时最短</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>限时费用最少</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>292</y>
     <width>253</width>
     <height>79</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="city0cbox">
        <property name="text">
         <string>北京</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="city1cbox">
        <property name="text">
         <string>上海</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="city2cbox">
        <property name="text">
         <string>重庆</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="city3cbox">
        <property name="text">
         <string>西安</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="city4cbox">
        <property name="text">
         <string>深圳</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="city5cbox">
        <property name="text">
         <string>郑州</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="city6cbox">
        <property name="text">
         <string>武汉</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="city7cbox">
        <property name="text">
         <string>拉萨</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="city8cbox">
        <property name="text">
         <string>海口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="city9cbox">
        <property name="text">
         <string>沈阳</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="city10cbox">
        <property name="text">
         <string>兰州</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="city11cbox">
        <property name="text">
         <string>昆明</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="SubWidget" name="LeftWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>211</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>策略要求：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>376</y>
     <width>281</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="TotalTimeLayout">
          <item>
           <widget class="QLabel" name="TotalTimeLabel">
            <property name="text">
             <string>总时间：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="TotalTime"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="TotalFareLayout">
          <item>
           <widget class="QLabel" name="CostLabel">
            <property name="text">
             <string>总费用：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Cost"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="PathList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>256</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="PathListContent">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>277</width>
         <height>197</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addTravelerButton">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>7</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>93</width>
     <height>28</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>93</width>
     <height>28</height>
    </size>
   </property>
   <property name="text">
    <string>添加旅客</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ThroughCityCheckBox">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>266</y>
     <width>147</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>是否选择途经城市</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>147</y>
     <width>161</width>
     <height>55</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="StartLabel">
        <property name="text">
         <string>起始地点：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="StartComboBox">
        <item>
         <property name="text">
          <string>北京</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>上海</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>重庆</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>西安</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>深圳</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>郑州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>武汉</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>拉萨</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>海口</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>沈阳</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>兰州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>昆明</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="DestinationLabel">
        <property name="text">
         <string>目标地点：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="DestinationComboBox">
        <item>
         <property name="text">
          <string>北京</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>上海</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>重庆</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>西安</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>深圳</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>郑州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>武汉</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>拉萨</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>海口</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>沈阳</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>兰州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>昆明</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>208</y>
     <width>77</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="StrategyLabel">
      <property name="text">
       <string>选择策略：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="DeadlineLabel">
      <property name="text">
       <string>限制时间：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>119</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>旅行线路：</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="StartDateTimeEdit">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>117</y>
     <width>158</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="DeadlineDateTimeEdit">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>237</y>
     <width>158</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>378</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>方案显示：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>7</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="TravelerChoseLabel">
      <property name="text">
       <string>选择旅客：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="TravelerComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>12</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>用户选项：</string>
   </property>
  </widget>
  <widget class="QLabel" name="StartLabel_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>起始时间：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>84</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查询状态</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>177</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>状态：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="at"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>48</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="SpentTimeLabel">
      <property name="text">
       <string>计时：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="SpentTime"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SubWidget</class>
   <extends>QWidget</extends>
   <header>SubWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
